admin/dump_all_version_info.py

```diff
@@ -1,7 +1,19 @@
+import os
 import sys
 
+from pip._internal.cli.main import main
+
+print("::group::Python info")
 print(sys.prefix)
 print(sys.exec_prefix)
 print(sys.executable)
 print(sys.version)
 print(sys.platform)
+print("::endgroup::")
+
+if os.environ.get("CI", "").lower() == "true":
+    # On CI show the exact deps found at runtime.
+    # Skipped on local dev to speed up the test execution.
+    print("::group::Deps info")
+    main(["freeze", "--all"])
```

Comment:
Just subprocess instead of using the private Python API?  Easy enough with `sys.executable`.

Reply:
I don't know why I don't like forking another process... I went for the entry_points API ... I hope this is considered public API :)

Reply:
Welp, I double checked that the entry point doesn't `sys.exit()` itself (at least not on success) so we do get the close tag for the group...  Hopefully they don't add another `console_script`.